holmes/plugins/toolsets/runbook/runbook_fetcher.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from typing import Any, Dict
+
+from holmes.core.tools import (
+    StructuredToolResult,
+    Tool,
+    ToolParameter,
+    ToolResultStatus,
+    Toolset,
+    ToolsetTag,
+)
+from holmes.plugins.runbooks import get_runbook_by_path
+
+
+# TODO(mainred): currently we support fetch runbooks hosted internally, in the future we may want to support fetching
+# runbooks from external sources as well.
+class RunbookFetcher(Tool):
+    toolset: "RunbookToolset"
+
+    def __init__(self, toolset: "RunbookToolset"):
+        super().__init__(
+            name="fetch_runbook",
+            description="Reads a runbook from the runbook catalog",
```

Comment:
I think a better description might be:
`Get runbook content by runbook path. Use this to get troubleshooting steps for incidents`